code/modules/projectiles/ammo_datums.dm

```diff
@@ -6,6 +6,8 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	var/name 		= "generic bullet"
 	var/icon 		= 'icons/obj/items/projectiles.dmi'
 	var/icon_state 	= "bullet"
+	var/handful_icon = "bullet" //used in icons/obj/items/ammo for use in generating handful sprites
```

Comment:
autodoc and name it icon_state since its not the icon
```suggestion
	///used in icons/obj/items/ammo for use in generating handful sprites
	var/handful_icon_state = "bullet" 
```